doc/aursearch.md

```diff
@@ -0,0 +1,29 @@
+# aursearch
+
+## Name
+
+aursearch - Search AUR package names by PCRE pattern
+
+## Synopsis
+
+aursearch [+-Fbqrv} [--] ARGS...
+
+## Description
+
+Search the AUR for packages. The argument is a PCRE pattern to search the AUR with.
+
+## Operations
+
+* __-F__ Search for a fixed string instead of a PCRE pattern
+* __-b__ Match pkgbase instead of pkgname
+* __-q__ Output less
+* __-r__ Output raw JSON from AUR RPC
```

Comment:
Nitpick: "raw JSON" is a tautology, I'd just say "Output JSON from [the] AUR RPC"